common/cross-profiles/aarch64-musl.sh

```diff
@@ -5,3 +5,5 @@ XBPS_CROSS_TRIPLET="aarch64-linux-musl"
 XBPS_CROSS_CFLAGS="-march=armv8-a"
 XBPS_CROSS_CXXFLAGS="$XBPS_CROSS_CFLAGS"
 XBPS_CROSS_FFLAGS=""
+XBPS_RUSTFLAGS="--sysroot=${XBPS_CROSS_BASE}/usr"
+XBPS_RUST_TARGET="aarch64-unknown-linux-musl""
```

Comment:
Should these also be `XBPS_CROSS` instead of `XBPS_`? Right now they're only required for cross anyway.